static/styles/rendered_markdown.css

```diff
@@ -309,8 +309,18 @@
         margin-bottom: 5px;
         margin-left: 5px;
         height: 100px;
-        display: block !important;
-        border: none !important;
+        width: 150px;
+        display: inline-block !important;
+        border: solid 1px transparent;
+        background: hsla(0, 0%, 0%, 0.1);
+        border-radius: 3px;
```

Comment:
How did we pick 3px here?  I mainly ask because we use 4px on most other images (avatars, etc.).

Reply:
Fellows from the [CZO thread](https://chat.zulip.org/#narrow/stream/101-design/topic/Images.20in.20a.20line.20vs.20row/near/1325988). Should we use 4px instead?

Reply:
Yeah, do 4px; I doubt Vlad was intending to specifically decide between 3px and 4px here.

Reply:
Done with 1711e88